cSpell.json

```diff
@@ -716,9 +716,8 @@
             ]
         },
         {
-            "filename": "**/specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json",
+            "filename": "**/specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/NetworkRuleSet-preview.json",
```

Comment:
This file doesn't exist in master branch.

Reply:
Right, looks like it got renamed again in #11743

Reply:
Looks like this is fixed now, so I removed it and added some related missing file-specific overrides that had gotten added to the root dictionary

Reply:
Did a little test script and cleaned out some other renamed files
```js
const fs = require('fs');

const cspell = JSON.parse(fs.readFileSync('cSpell.json', 'utf-8'));

cspell.overrides.forEach(element => {
    let filename = element.filename.substring(3);

    try {
        fs.statSync(filename)
    } catch (error) {
        console.warn(`BAD: ${filename}\n${error}`)
    }
});
```